scripts/go.mod

```diff
@@ -6,21 +6,23 @@ replace github.com/nspcc-dev/neo-go => ../
 
 require (
 	github.com/davecgh/go-spew v1.1.1
-	github.com/nspcc-dev/neo-go v0.0.0-00010101000000-000000000000
+	github.com/nspcc-dev/neo-go v0.106.0
```

Comment:
106.3?

Reply:
We have replace derictive here anyway.

Reply:
`replace` is good. That's why this version doesn't matter much. But still if you're changing it better be current.

Reply:
Fixed.